boards/common/particle-mesh/Makefile.include

```diff
@@ -9,18 +9,38 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 # add the common header files to the include path
 INCLUDES += -I$(RIOTBOARD)/common/particle-mesh/include
 
-# This board uses a DAP-Link programmer
-# Flashing support is provided through pyocd (default) and openocd.
-# For openocd, a version built against the development branch and containing
-# the support for nrf52 cpu is required.
-PROGRAMMER ?= pyocd
-ifeq (pyocd,$(PROGRAMMER))
-  # The board is not recognized automatically by pyocd, so the CPU target
-  # option is passed explicitly
-  FLASH_TARGET_TYPE ?= -t nrf52840
-  include $(RIOTMAKE)/tools/pyocd.inc.mk
-else ifeq (openocd,$(PROGRAMMER))
-  DEBUG_ADAPTER ?= dap
+ifeq (1,$(PARTICLE_MONOFIRMWARE))
+  CFLAGS += -DPARTICLE_MONOFIRMWARE
+  ROM_OFFSET = 0x30000
+  FW_ROM_LEN = 0xc4000
+  FLASHFILE = $(BINFILE)-checksummed
+  # Setting DFU_ARGS won't work as the implied --reset causes errors.
+  FFLAGS = -d 0x2B04:0xD00E -a 0 -s 0x30000:leave -D $(FLASHFILE)
+  PROGRAMMER = dfu-util
+  include $(RIOTMAKE)/tools/dfu.inc.mk
+
+  USEMODULE += usb_board_reset
```

Comment:
You want to also

    include $(RIOTMAKE)/tools/usb_board_reset.mk

Reply:
Doesn't this only apply if cdc_acm_stdio is also active?

By default it isn't, and then the USB board reset mechanism doesn't apply. (I *do* think it should be default anyway here, but that's not what I'm fixing here; when it was in earlier versions of this, I was reaching too far for a single PR).

(And btw since the last fixup, the USEMODULE is in .dep where it should be according to murdock)

Reply:
> Doesn't this only apply if cdc_acm_stdio is also active?

You are right - I just checked the other boards though and they will always select it if their USB bootloader is selected as `$(PROGRAMMER)`.
That is not always correct. (but then again, there is no other reset method if you are using the usb bootloader, so at worst it will just not work)

> when it was in earlier versions of this, I was reaching too far for a single PR

fortunately now all you need to do is 

    include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk

Reply:
I've given it a try anyway -- they do fall through safely, so even if CDC-ACM is *not* active, all the ill effect they give is a few "failed to" lines during the build, but it still tries to go for the DFU later, and that's then where the user has to intervene. Fixup'd.

Reply:
Waah, asynchronous communication :-)

It's still something completely different than what is in this PR (if nothing else, it'd be missing from the changelogs if changed here), so if it's all the same to you, I'd leave that as is up to the point of having included the `usb_board_reset.mk` (for it sure is newly available), and will take the steps you neatly laid out to a separate PR.

Reply:
Ok if you want to add stdio_cdc_acm in a separate PR that's also fine.